README.md

```diff
@@ -23,12 +22,14 @@ A cli to browse and watch anime. This tool scrapes the site [gogoanime](https://
 ## Usage
 
   ```text
-    ani-cli [-v | -i] [-q <quality>] [-d | -p <download_dir>] [<query>]
-    ani-cli [-v | -i] [-q <quality>] -c
+    ani-cli [-v | -i] [-q <quality>] -e [<arguments>] [-d | -p <download_dir>] [<query>]
```

Comment:
Does is pass arguments to aria2c in this case?

Reply:
It doesn't. Should I add it?

Reply:
No, not needed but in that case don't add it to that line